spec/requests/page_views_spec.rb

```diff
@@ -4,11 +4,12 @@
   let(:user) { create(:user, :trusted) }
   let(:article) { create(:article) }
 
-  before do
-    # rubocop:disable RSpec/AnyInstance
-    allow_any_instance_of(PageViewsController).to receive(:skip_page_view_update?).and_return(false)
-    allow_any_instance_of(PageViewsController).to receive(:skip_organic_page_view_update?).and_return(false)
-    # rubocop:enable RSpec/AnyInstance
+  def page_views_post(params)
+    sidekiq_perform_enqueued_jobs do
+      post "/page_views", params: params
```

Comment:
I tried to use `page_views_path` instead, but for two specific specs (the ones starting at lines 81 and 88 respectively) this resulted in weird errors in `sidekiq_perform_enqueued_jobs` (` NoMethodError: undefined method `url_options'`) and I really didn't feel like spending more time on debugging this right now when there's a workable alternative.